client/src/components/Button.tsx

```diff
@@ -0,0 +1,26 @@
+import { ButtonProps, Button as BootstrapButton } from "react-bootstrap";
+import { useNavigate } from "react-router-dom";
+import React from "react";
+
+// Button wrapper for react-router
+// Using hrefs lowers performance
+
+function Button({ href, onClick, ...oP }: ButtonProps) {
```

Comment:
What is oP?

Reply:
other props, to be passed to the bootstrap button.

Reply:
could we change it to `otherProps` then?

Reply:
sure